frontend/src/main/components/AliasApprovalTable.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+import { useBackendMutation } from "main/utils/useBackend";
+import { toast } from "react-toastify";
+import { useQueryClient } from "react-query";
+
+export default function AliasApprovalTable({ users }) {
+  const testid = "AliasApprovalTable";
+  const queryClient = useQueryClient();
+
+  const objectToAxiosParams = (variables) => {
+    if (
+      !variables ||
+      typeof variables.id === "undefined" ||
+      typeof variables.approved === "undefined"
+    ) {
+      console.error("Missing id or approved in mutation variables:", variables);
+      return {};
+    }
+    return {
+      url: "/api/currentUser/updateAliasModeration",
+      method: "PUT",
+      params: {
+        id: variables.id,
+        approved: variables.approved,
+      },
+    };
+  };
+
+  const onSuccess = (returnedUser, variables) => {
+    toast(
+      `${variables.approved ? "Approved" : "Rejected"} alias: ${returnedUser.alias}`,
+    );
+    queryClient.invalidateQueries("alias-approval");
+  };
+
+  const mutation = useBackendMutation(objectToAxiosParams, { onSuccess }, [
+    "alias-approval",
```

Comment:
```suggestion
   ["/api/admin/usersWithProposedAlias"],
```

This needs to match the key that you use in the `useBackend` call.  The mismatch is the reason that the table doesn't refresh when you approve or reject an alias.